spec/requests/users_controller_spec.rb

```diff
@@ -123,6 +130,10 @@
           expect(page).not_to have_link("Deactivate user", href: "/users/#{user.id}/deactivate")
         end
 
+        it "does not allow resending invitation emails" do
+          expect(page).not_to have_link("Resend invite link", href: "/users/#{other_user.id}/resend-invite")
```

Comment:
There are a lot of unhappy path tests here, but do you have one that ensures the confirmation method is called for a given user?